e2e/scenarios/kops-simple

```diff
@@ -120,13 +148,6 @@ if [[ -n "${SSH_PRIVATE_KEY:-}" ]]; then
   KUBETEST2_ARGS="${KUBETEST2_ARGS} --ssh-private-key=${SSH_PRIVATE_KEY}"
 fi
 
-function delete-cluster {
-    # shellcheck disable=SC2153
-  if [[ "${DELETE_CLUSTER:-}" == "true" ]]; then
-      kubetest2 ${KUBETEST2_ARGS} --down || echo "kubetest2 down failed"
-  fi
-}
-trap delete-cluster EXIT
```

Comment:
On line 160 you have

>   --kubernetes-version="${K8S_VERSION}" \

is that K8S_VERSION relevant? because I see it is defaulted to a versions we are not going to use
```
if [[ -z "${K8S_VERSION:-}" ]]; then
  K8S_VERSION="$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)"
fi
```

Reply:
Good point.  So this is the version of k/k itself (kube-apiserver, kubelet, kube-controller-manager etc) that we are installing onto the cluster.  Also the version of  the k/k test-suite we are going to run.

I suspect for this particular test scenario we probably want to either build k/k from source as well, or we want to just run the latest.  We could also try to figure out the prebuilt version which corresponds to the version of kubernetes that is checked out, but that seems hard!

Reply:
from the top of my head I can't see a scenario that cloud-provider-gcp vendors a k8s versions higher than the one it will be running against ... I think that for testing this should match the one in the WORKSPACE